src/pages/components/tile/usage.mdx

```diff
@@ -587,7 +572,7 @@ base tiles.
 
 <Caption>
   Clickable, expandable, and selectable tiles are clickable anywhere within the
-  tile.
+  tile
```

Comment:
tile. 

[Always put periods at the end of complete sentences. Sentence fragments don't get a periods.]